### DISCUSSION TITLE
Does Visp has an example for simulating eye-to-hand visual servo?

### DISCUSSION THREAD

**Discussion by Steven-ZY** [lagadic/visp#1240] [Q&A]
Does Visp has an example for simulating eye-to-hand visual servo?

**Comment by fspindle** [comment]
The only example I can point is the one used on our gantry robot in [servoAfma6Points2DCamVelocityEyeToHand.cpp](https://github.com/lagadic/visp/blob/master/example/servo-afma6/servoAfma6Points2DCamVelocityEyeToHand.cpp) from which you can get inspiration
